tests/features/common/kie-kieserver-common.feature

```diff
@@ -19,14 +19,14 @@ Feature: Kie Server common features
     When container is started with env
       | variable         | value       |
       | KIE_ADMIN_ROLES  | kie-server  |
-      | KIE_ADMIN_PWD    | kieserver1! |
+      | KIE_ADMIN_PWD    | kieserver$0 |
```

Comment:
Change of this test is not needed, right? It's covered by new scenario for RHPAM-3211

Reply:
Reverted